modules/doc/content/newsletter/2022_01.md

```diff
@@ -7,6 +7,25 @@ for a complete description of all MOOSE changes.
 
 ## MOOSE Improvements
 
+### PeripheralTriangleMeshGenerator
+
+A new mesh generator available in the MOOSE reactors module, the
+[PeripheralTriangleMeshGenerator.md], allows users to take an arbitrary
+2-D input mesh (typically a reactor core) and
+automatically add a triangulation of the area in between the input
+mesh boundary and a circular periphery.  For thick peripheries,
+element size within that area can be reduced by user-specified
+parameters defining extra circles of element vertices to add.  The
+output mesh may then be passed to other generators for further
+extension, extrusion to 3-D, etc.
+
+[Figure 1](#triangle_steiner) shows an example triangulation around an existing core mesh.
+
+!media media/meshgenerators/triangle_abtr_steiner.png
```

Comment:
```suggestion
!media modules/reactor/doc/content/media/meshgenerators/triangle_abtr_steiner.png
```
One comment on this - is there any way you can use the original copy of this file in the reactor module rather than duplicating the file in `modules/doc`? We build all modules when building the full website docs, sot it should be available.

Reply:
I'd love to, but if I interpret your question literally as a question then the answer is "I don't know".  I'm not familiar with this branch of markdown, and `find -name '*.md' | xargs grep '!media' | grep '\.\.'` came up blank for me so I was afraid using relative path names might not be possible.

Reply:
The way MooseDocs determines the available content is a bit weird. It needs to know that a requested file has been committed (it develops a file list using git), but before that it needs to know what locations are considered valid content. See [the link here](https://github.com/idaholab/moose/blob/ecbae1a77c12ff8dfe1af01e13f9b736857b7ba4/modules/doc/config.yml#L21) showing that we include the `modules/reactor/doc/content` folder in the main `modules/doc` documentation config for the website. Once MooseDocs knows what files are available, it copies what it needs to a website file structure so that everything is where the rendered webpages expect them to be (hence the truncated file path.....MooseDocs will look for files based on the "file roots" listed in the Content config). 

My suggestion above might actually be a poor one....You might only need to remove the `triangle_abtr_steiner.png` file you added in the `modules/doc/content/media` folder, and the build system will find the other one you already had added before. I should test this locally real quick before I ask you to make any changes (so as not to re-run every recipe before we need to).

Reply:
Removed my code suggestion. I confirmed that if you just remove the file `modules/reactor/doc/content/media/meshgenerators/triangle_abtr_steiner.png`, the docs will build successfully using the copy in `modules/reactor`.

Reply:
Just saw your new commit - everything looks good otherwise. Thanks @roystgnr!